### ISSUE TITLE
rs-ws-02-2

### ISSUE THREAD

**Issue by tnauss** [logmoc/msc-phygeo-class-of-2016-Heimannc#2]
Insgesamt gut gelöst. Sie haben aber die (nicht mehr benötigten) Ausgangsdateien nicht umbenannt. Außerdem fehlt die Visualisierung.

**Comment by Heimannc** [comment]
Vielen Dank für Ihre Rückmeldung,

dummerweise haben wir tatsächlich bei der Bearbeitung der Aufgabe den letzten Abschnitt mit der Visualisierung überlesen.
Wäre es diesbezüglich möglich dies und die Umbenennung der alten Ausgangsdateien kurz nachzuholen und die Lösung im Anschluss erneut hochzuladen?

**Comment by tnauss** [comment]
Ja, können sie machen (an der Note ändert es nichts - wir haben dafür 
sowieso kaum was abgezogen und es ist immer noch eine 1).

Viele Grüße

Thomas Nauss

**Comment by Heimannc** [comment]
Nun die überarbeitete Aufgabe inklusive Visualisierung und Umbenennung der Ausgangsdateien!

Viele Grüße
